### ISSUE TITLE
Metric symbol doesn't get exported on Product Export

### ISSUE THREAD

**Issue by wakqasahmed** [akeneo/pim-community-dev#5750]
**I'm reporting a Bug/Feature**

Confirmed at http://demo.akeneo.com as well.

In the product export, metric unit is being exported but no way to have symbol (e.g. `KILOGRAM` is exported but we may require `kg`, `SQUARE_MILLIMETER` is exported but we may require `mm²`)

Looking at MeasureManager (https://github.com/akeneo/pim-community-dev/blob/d79551a73994fc848e84df7aa3d04eb1cf90b0d9/src/Akeneo/Bundle/MeasureBundle/Manager/MeasureManager.php) there is no method to `getSymbolByUnit` and no configuration available to say whether to export unit or symbol while Product export.

Also, there is translation issue coz unit code is saved in database while exporting usually require translated symbol to be exported.

Any idea how to handle the above situations?

**Comment by mickaelandrieu** [comment]
Hello @wakqasahmed, thank you for your feedback.

I agree this is a bug and I can reproduce it, thanks for explanations!

Would you like to contribute to a patch? Let me know if you need help!

Regards

**Comment by a2xchip** [comment]
Hello @wakqasahmed @mickaelandrieu 
First of all thank you @wakqasahmed that you raised this issue.

Could you guys confirm that something else but not measurement unit code was used in 1.6 during export? As i remember it was same as now - code. If so it should not be labeled as bug but like feature request. And feature as i understand is to have unit name or symbol to be exported in a line with measurement unit code.
 
;-) Thanks!

**Comment by mickaelandrieu** [comment]
@a2xchip I agree this is the actual (and current behavior) but as it creates translation errors, it should be regarded as an issue, isn't it?

Also, it's weird to not be able to export/import a "complete" unit of measurement.

I ask to the product team and I come back with more information later :)

**Comment by mickaelandrieu** [comment]
I'm back with more information.

We are aware of this missing feature and if you want to contribute to this feature we will accept it (including the side effect on translation issues).

What do you think @wakqasahmed? Can you help on it?

Regards,

**Comment by wakqasahmed** [comment]
My apology, I couldn't contribute back then. Is it resolved in the newer version? @a2xchip @mickaelandrieu

**Comment by a2xchip** [comment]
@LaureBro

**Comment by SamirBoulil** [comment]
Hello @wakqasahmed!

The product export has not changed in newer versions regarding your issue. 

However, one way to get a unit symbol is through the API (See for the SAAS version https://api.akeneo.com/api-reference.html#measurement_families_get_list and for the 4.0 version https://api.akeneo.com/api-reference.html#measure_families_get_list).

I hope this helps!

**Comment by nbogucki** [comment]
Hello,
I did a pull request last week, which resolved this issue https://github.com/akeneo/pim-community-dev/pull/18709 .
Can you tell me how long do I need to wait for a code review?

**Comment by a2xchip** [comment]
@nidup
